Help installing new app

Hi there,

I am receiving the following when I try to setup this repo today (I have had success in the past).

I am using node 16.13.2 on an up to date M1 Mac.

```npx gatsby new starter-ts https://github.com/jpedroschmitz/gatsby-starter-ts
info Creating new site from git: https://github.com/jpedroschmitz/gatsby-starter-ts.git

Cloning into 'starter-ts'...
remote: Enumerating objects: 43, done.
remote: Counting objects: 100% (43/43), done.
remote: Compressing objects: 100% (33/33), done.
remote: Total 43 (delta 1), reused 35 (delta 1), pack-reused 0
Receiving objects: 100% (43/43), 264.77 KiB | 6.02 MiB/s, done.
success Created starter directory layout
info Installing packages...

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: gatsby-starter-ts@1.0.0
npm ERR! Found: react@18.1.0
npm ERR! node_modules/react
npm ERR!   react@"18.1.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.9.0 || ^17.0.0" from @testing-library/react-hooks@8.0.0
npm ERR! node_modules/@testing-library/react-hooks
npm ERR!   dev @testing-library/react-hooks@"8.0.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/nick/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:

 ERROR

Command failed with exit code 1: npm install
```

I noticed the @testing-library/react-hooks docs has the following note.

```NOTE: The minimum supported version of react, react-test-renderer and react-dom is ^16.9.0``` 

Am I doing something wrong ? Thanks!

--- a/.npmrc
+++ b/.npmrc
@@ -1,1 +1,2 @@
 save-exact = true
+legacy-peer-deps=true

